contrib/yash/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "yash"
+pkgver = "2.56.1"
+pkgrel = 0
+build_style = "configure"
```

Comment:
gnu_configure + `configure_gen = []`

Reply:
I can't otherwise I get this error:
```
/builddir/yash-2.56.1/configure: --sysconfdir=/etc: unknown argument
```

yash's configure will return 1 if it doesn't recognize an argument:
https://github.com/magicant/yash/blob/trunk/configure#L156

Reply:
patch out the error

Reply:
hm, or actually maybe don't... doesn't seem that beneficial here, let's keep configure